src/app/lecture/knowledge-node/knowledge-node.component.html

```diff
@@ -22,29 +22,30 @@ <h3>Goal: {{knowledgeNode.learningObjective}}</h3>
         *ngFor="let learningObject of knowledgeNode.learningObjects"
         [learningObject]="learningObject"></cc-learning-object-container>
       <br>
+      <cc-feedback (submitRating)="submitRating($event)"></cc-feedback>
```

Comment:
We need to check with @vdragan1993 if we want feedback for whole knowledge node or individual learning objects for collab. filtering.

We can also have the feedback component receive the KN/LO id as input, and perform the HTTP request by itself (e.g., directly inject the feedbackService into it (and package it alongside the component).

Reply:
@milicasiriski Dragan confirmed - we should tie the component to each LearningObject.